lighthouse-core/lib/asset-saver.js

```diff
@@ -19,6 +19,8 @@ const LHError = require('../lib/lh-error.js');
 // TODO(esmodules): Rollup does not support `promisfy` or `stream.pipeline`. Bundled files
 // don't need anything in this file except for `stringifyReplacer`, so a check for
 // truthiness before using is enough.
+// TODO: Can remove promisify(pipeline) in Node 15.
+// https://nodejs.org/api/stream.html#streams-promises-api
```

Comment:
just moving the TODO to see it more easily with the other hacky workaround that can eventually be removed